### PR TITLE
Added just the energy preferences xml.

### PR DESCRIPTION
**Pull request by drempelg**
Added just the xml and regenerated.

No implementations yet as requested.

### CODE REVIEW

**Review by andy31415 — APPROVED** [review]

File: src/app/zap-templates/zcl/data-model/chip/energy-preference-cluster.xml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    <server init="false" tick="false">true</server>
+    <description>This cluster provides an interface to specify preferences for how devices should consume energy.</description>
+    <!--Attributes-->
+
+    <!--Conformance feature BALA - for now optional-->
+    <attribute code="0x0000" side="server" type="ARRAY" entryType="BalanceStruct" define="ENERGY_PREFERENCE_ENERGY_BALANCES" isNullable="false" min="2" max="10" writable="false" optional="true">EnergyBalances</attribute>
+
+    <!--Conformance feature BALA - for now optional-->
+    <attribute code="0x0001" side="server" type="int8u" define="ENERGY_PREFERENCE_CURRENT_ENERGY_BALANCE" isNullable="false" writable="true" optional="true">
+      <access op="read" privilege="view"/>
```
Comment (andy31415) [inline src/app/zap-templates/zcl/data-model/chip/energy-preference-cluster.xml:31]: Nit: I think thiese are the default if not specified. 7.6 says:

```
Attributes, commands, and events that do not define any privileges as access qualities SHALL be deemed to have the following:
   - View privilege required for Read access,
   - Operate privilege required for Write access,
   - Operate privilege required for Invoke access for request commands.
```

File: src/app/zap-templates/zcl/data-model/chip/energy-preference-cluster.xml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    <domain>Energy Management</domain>
+    <code>0x009B</code>
+    <define>ENERGY_PREFERENCE_CLUSTER</define>
+    <client init="false" tick="false">true</client>
+    <server init="false" tick="false">true</server>
+    <description>This cluster provides an interface to specify preferences for how devices should consume energy.</description>
+    <!--Attributes-->
+
+    <!--Conformance feature BALA - for now optional-->
+    <attribute code="0x0000" side="server" type="ARRAY" entryType="BalanceStruct" define="ENERGY_PREFERENCE_ENERGY_BALANCES" isNullable="false" min="2" max="10" writable="false" optional="true">EnergyBalances</attribute>
```
Comment (bzbarsky-apple) [inline src/app/zap-templates/zcl/data-model/chip/energy-preference-cluster.xml:27]:
```suggestion
    <attribute code="0x0000" side="server" type="array" entryType="BalanceStruct" define="ENERGY_PREFERENCE_ENERGY_BALANCES" isNullable="false" min="2" max="10" writable="false" optional="true">EnergyBalances</attribute>
```
Everything consistently uses lowercase built-in names now, except some of the new media bits.

File: src/app/zap-templates/zcl/data-model/chip/energy-preference-cluster.xml [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+    <!--Conformance feature BALA - for now optional-->
+    <attribute code="0x0001" side="server" type="int8u" define="ENERGY_PREFERENCE_CURRENT_ENERGY_BALANCE" isNullable="false" writable="true" optional="true">
+      <access op="read" privilege="view"/>
+      <access op="write" privilege="operate"/>
+      <description>CurrentEnergyBalance</description>
+    </attribute>
+
+    <!--Conformance feature BALA - for now optional-->
+    <attribute code="0x0002" side="server" type="ARRAY" entryType="EnergyPriorityEnum" define="ENERGY_PREFERENCE_ENERGY_PRIORITIES" isNullable="false" max="2" writable="false" optional="true">EnergyPriorities</attribute>
```
Comment (bzbarsky-apple) [inline src/app/zap-templates/zcl/data-model/chip/energy-preference-cluster.xml:37]:
```suggestion
    <attribute code="0x0002" side="server" type="array" entryType="EnergyPriorityEnum" define="ENERGY_PREFERENCE_ENERGY_PRIORITIES" isNullable="false" max="2" writable="false" optional="true">EnergyPriorities</attribute>
```

File: src/app/zap-templates/zcl/data-model/chip/energy-preference-cluster.xml [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+    <!--Conformance feature LPMS - for now optional-->
+    <attribute code="0x0003" side="server" type="ARRAY" entryType="BalanceStruct" define="ENERGY_PREFERENCE_LOW_POWER_MODE_SENSITIVITIES" isNullable="false" min="2" max="10" writable="false" optional="true">LowPowerModeSensitivities</attribute>
```
Comment (bzbarsky-apple) [inline src/app/zap-templates/zcl/data-model/chip/energy-preference-cluster.xml:40]:
```suggestion
    <attribute code="0x0003" side="server" type="array" entryType="BalanceStruct" define="ENERGY_PREFERENCE_LOW_POWER_MODE_SENSITIVITIES" isNullable="false" min="2" max="10" writable="false" optional="true">LowPowerModeSensitivities</attribute>
```

**Review by bzbarsky-apple — APPROVED** [review]

**Pull request merged**